build_projects/update-dependencies/project.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "NETStandard.Library": "1.6.0-rc3-24201-00",
     "Microsoft.CSharp": "4.0.1-rc3-24201-00",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24201-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24201-00",
```

Comment:
Lets try to avoid Microsoft.NETCore.App dependencies in the repo that builds Microsoft.NETCore.App. It will be easier that way.